lib/services.cf

```diff
@@ -118,10 +131,16 @@ bundle agent standard_services(service,state)
 # methods:
 #     "" usebundle => standard_services("sshd", "start"); # direct
 # ```
+#
+# Alternatively, since services promises are an abstraction around bundles, the service state can be promised via a methods type promise.
+#
+# ```cf3
+#
+# methods:
+#     "SSHD should be running" usebundle => standard_services("sshd", "start");
```

Comment:
Why would you want to do this? Other than that it is technically possible?

Reply:
it /might/ be preferable because of difference in normal ordering. But generally, you woudlnt do this because it's mroe clear to use the services syntax.

Reply:
so maybe add the why to this comment so people don't just use the pattern.